SECURITY.md

```diff
@@ -0,0 +1,175 @@
+policy sets out our definition of good faith in the context of finding and
+reporting vulnerabilities, as well as what you can expect from us in return.
+
+# Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Extend Safe Harbor for your vulnerability research that is related to this
+  policy;
+- Work with you to understand and validate your report, including a timely
+  initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner;
+- Publicly disclose accepted vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to
+  report a unique vulnerability, and your report triggers a code or
+  configuration change.
+
+TODO: Is Knative Security up for including the recognize clause? Do we have a
+page for this?
+
+# Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between
+good-faith hacking and malicious attack, we ask that you:
+
+- Play by the rules. This includes following this policy, as well as any other
+  relevant agreements. If there is any inconsistency between this policy and any
+  other relevant terms, the terms of this policy will prevail;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting out-of-scope systems,
+  destroying data, and/or harming user experience;
+- Use only the [Official Channels](#reporting-a-vulnerability) to discuss
+  vulnerability information with us;
+- Keep the details of any discovered vulnerabilities confidential until they are
+  fixed, according to the Disclosure Policy;
+- Perform testing only on in-scope systems, and respect systems and activities
+  which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of
+  data you access to the minimum required for effectively demonstrating a Proof
+  of Concept; and cease testing and submit a report immediately if you encounter
+  any user data during testing, such as Personally Identifiable Information
+  (PII), Personal Healthcare Information (PHI), credit card data, or proprietary
+  information;
+- You should only interact with test accounts and Knative instances you own or
+  with explicit permission from the account holder; and
+- Do not engage in extortion.
+
+# Safe Harbor
+
+When conducting vulnerability research according to this policy, we consider
+this research to be:
+
+- Authorized in accordance with the Computer Fraud and Abuse Act (CFAA) (and/or
+  similar state laws), and we will not initiate or support legal action against
+  you for accidental, good faith violations of this policy;
+- Exempt from the Digital Millennium Copyright Act (DMCA), and we will not bring
+  a claim against you for circumvention of technology controls;
+- Exempt from restrictions in our Terms & Conditions that would interfere with
```

Comment:
Do we have "Terms & Conditions", or is this lifted from some product context?

Reply:
This file is adapted from https://github.com/disclose/terms/blob/master/regional/USA-core-terms.md

The license file refers to "terms and conditions": https://github.com/knative/docs/blob/master/LICENSE

The knative.dev website has some policy links at the bottom. Interestingly, the "privacy policy" one links to Google's: https://policies.google.com/privacy.